_hackbars/ties_klinkhamer.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: any_title
+author: your_name
+---
+
+<!-- Your MESSAGE on open source-->
```

Comment:
hey @tiesklinkhamer Can you please add `title`, `author` fields and add your MESSAGE on climate change?

Reply:
Also your MESSAGE on open source